ansible/roles_ocp_workloads/ocp4_workload_pntae/defaults/main.yml

```diff
@@ -3,19 +3,5 @@ become_override: False
 ocp_username: ablum-redhat.com
 silent: False
 
-# If your workload needs customization options provide variables to be used.
-# Variable names must be prefixed by the role name "<role name>_<variable>".
-# Therefore because this example workload is called "ocp4_workload_example" the variables
-# should be named "ocp4_workload_example_variable".
-#
-# You can override the defaults as parameters to the Ansible run that runs
-# your workload.
-
-# Documentation for variable 1 (what does it do?)
-#ocp4_workload_example_variable_1: "value 1"
-
-# Documentation for variable 2 (what does it do?)
-#ocp4_workload_example_variable_2: "value 2"
-
 # PnTAE course id should match the name of the grading script, catalog item, and LMS course_code
-ocp4_workload_example_pntae_course_id: "cee-cf-110"
+ocp4_workload_example_course_id: "cee-cf-110"
```

Comment:
You got it right in the other two files... this one also has to be `ocp4_workload_pntae_course_id`